### PR TITLE
B2C Implementation

### PR DESCRIPTION
**Pull request by TraceyOnim**
## What is the Purpose?

Briefly describe what the PR addresses

## What was the approach?

Briefly describe the approach used to address the issue

## Are there any concerns to addressed further before or after merging this PR?

State some additional info if any. For instance running `mix deps.get` or setting some environment variable(s)

## Mentions?

Mention persons you'd like to review this PR

## Issue(s) affected?

List of issues addressed by this PR.

### CODE REVIEW

File: lib/ex_pesa/Mpesa/b2c.ex [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+  import ExPesa.Mpesa.MpesaBase
+
+  @doc """
+  Initiates a B2C mpesa request.
+
+
+  """
+
+  def initiate_request(%{
```
Comment (manuelgeek) [inline lib/ex_pesa/Mpesa/b2c.ex:18]: @TraceyOnim rename this to request from initiate request

**Comment by manuelgeek** [comment]
@TraceyOnim I left few comments above, also remember to update the `config/dev.sample.ex` with the keys added

**Comment by manuelgeek** [comment]
@TraceyOnim  @sigu  Kindly finish on the securityCredential generation, the method already exists in utils.ex, reuse it.
I with we close this over weekend, we're doing a release early next week

**Comment by lenileiro** [comment]
@TraceyOnim @sigu @manuelgeek I think this is ready to ship

**Review by lenileiro — DISMISSED** [review]
LGTM

**Comment by manuelgeek** [comment]
@lenileiro  or @TraceyOnim  kindly rebase, this fixes the pipelines issue
again @TraceyOnim you can talk to @lenileiro and the issues you were facing yesterday

**Comment by manuelgeek** [comment]
@lenileiro @TraceyOnim  there's a conflict

**Review by manuelgeek — APPROVED** [review]
finally it's here, looks OK now,

**Pull request merged**